source/gloperate/source/painter/PipelineOutputCapability.cpp

```diff
@@ -0,0 +1,36 @@
+#include <gloperate/painter/PipelineOutputCapability.h>
+
+#include <gloperate/base/collection.hpp>
+
+#include <gloperate/pipeline/AbstractData.h>
```

Comment:
```AbstractData```, ```Stage``` and ```Data``` includes are not necessary here since you don't resolve any members of those classes in this ```cpp``` file.